sandboxed_containers/sandboxed-containers-release-notes.adoc

```diff
@@ -0,0 +1,171 @@
+
+* Previously, the `SourceImage` field was displayed on the web console and using the field had no effect on the installation. With this release, the unused `SourceImage` field is no longer displayed when creating the `KataConfig` custom resource from the web console. (link:https://issues.redhat.com/browse/KATA-1015[*KATA-1015*])
+
+[id="sandboxed-containers-1-2-known-issues"]
+== Known issues
+
+* If you are using {sandboxed-containers-first}, you might receive SELinux denials accessing files or directories mounted from the `hostPath` volume in an {product-title} cluster. These denials can occur even when running privileged sandboxed containers, since privileged sandboxed containers do not disable SELinux checks.
++
+Following SELinux policy on the host guarantees full isolation of the host file system from the sandboxed workload by default, and provides stronger protection against potential security flaws in `virtiofsd` or QEMU.
++
+If the mounted files or directories do not have specific SELinux requirements on the host, you can use local persistent volumes as an alternative. Files are automatically relabeled to `container_file_t`, following SELinux policy for container runtimes. See xref:../storage/persistent_storage/persistent-storage-local.adoc[Persistent storage using local volumes] for more information.
++
+Automatic relabeling is not an option when mounted files or directories are expected to have specific SELinux labels on the host. Instead, you can set custom SELinux rules on the host in order to allow virtiofsd to access these specific labels. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1904609[*BZ#1904609*])
+
+* You might encounter an issue where `machine-config-operator` (MCO) pod changes to `CrashLoopBackOff` state, and the "openshift.io/scc" annotation of the POD shows "sandboxed-containers-operator-scc" instead of the default "hostmount-anyuid".
++
+If you encounter this issue, perform the following steps:
++
+. Change the `selinuxOptions` strategy by running the following command:
++
+[source,terminal]
+----
+oc patch scc sandboxed-containers-operator-scc --type=merge --patch '{"seLinuxContext":{"type": "RunAsAny"}}'
+----
+
+. Scale a deployment by running the following command:
++
+[source,terminal]
+----
+oc scale deployments/machine-config-operator -n openshift-machine-config-operator --replicas=0
+----
+
+. Scale a deployment again by running the following command:
++
+[source,terminal]
+----
+oc scale deployments/machine-config-operator -n openshift-machine-config-operator --replicas=1
+----
+
+. Change the `selinuxOptions` strategy by running the following command:
```

Comment:
Revert the 'seLinuxOptions' strategy of the "sandboxed-containers-operator-scc" to its original value of 'MustRunAs' by running the following command: